### DISCUSSION TITLE
Publish Flutter iOS app to TestFlight

### DISCUSSION THREAD

**Discussion by icarusoft** [codemagic-ci-cd/codemagic-docs#926] [Q&A] [answered]
Hi,

I am not experienced with App Store Connect and TestFlight. I have access to a App on App Store Connect (I am not the owner or manager), and have configured Codemagic with correct App Store Connect Api key and so on. As I understand it, I need to publish to App Store Connect first to be able to publish to TestFlight, right?

I have successfully made a iOS build on Codemagic, but publish to App Store Connect fails with the following error:

```
Did not find build matching uploaded version yet, it might be still processing. Waiting to try again...
The application binary was successfully uploaded to App Store Connect for processing but the corresponding build is not available in App Store Connect. This usually happens when the uploaded application has one or more issues. Please check your email for direct feedback from Apple
```

I am not receiving any emails from Apple, maybe they go to someone else?

Any ideas on what to try to fix this?

Best regards

**Comment by K-Codemagic** [comment, marked as answer]
Seems to be an issue with the version, can you please share the build ID so I can investigate further?

**Reply by icarusoft** [reply]
630f56a62923d8e50dba3af3
and
630f6b3b2923d8b80da05d66

**Reply by K-Codemagic** [reply]
Hi, this is related to post-processing, which is Apple related. You should have received an email from Apple about what issues your app could have. Can you please check the Apple registered email?

**Reply by icarusoft** [reply]
It was a wrong version code, I have fixed it now.

**Reply by K-Codemagic** [reply]
Great, have a nice day.
